data/data/hra-growth.vl.json

```diff
@@ -33,12 +37,22 @@
     }
   ],
   "encoding": {
-    "x": { "timeUnit": "yearmonth", "field": "date", "title": "Date" },
+    "x": {
+      "timeUnit": "yearmonth",
+      "field": "date",
+      "title": "Date",
+      "axis": {
+        "labelAngle": -45
```

Comment:
Do we need to tilt for this vis? Looking at the current 4:3 version the x axis labels fit without tilting.

Reply:
When I talked to libby this morning, she said we wanted the labels to be tilted for this vis.